app/react/Documents/components/scss/showToc.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../../App/scss/config/_colors.scss";
+
+.toc {
+  padding-left: 5px !important;
+  padding-right: 5px !important;
+
+  ul.toc-view {
+    $indents: 5;
+
+    @for $i from 0 through $indents {
```

Comment:
Not really sure the usefulness of a variable with fixed value used in the very next line?  `from 0 through 5` should be enough?

Reply:
@grafitto can you check this please ?

Reply:
Removed the loop completely